<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Andrey Rijov</author>
 <class>Copy_VM_Window</class>
 <widget class="QDialog" name="Copy_VM_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Copy VM</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/icons.qrc">
    <normaloff>:/copy.png</normaloff>:/copy.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="Label_Name">
     <property name="text">
      <string>&amp;New VM name:</string>
     </property>
     <property name="buddy">
      <cstring>Edit_New_VM_Name</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="Edit_New_VM_Name"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="CH_Copy_Disk_Images">
     <property name="text">
      <string>Copy disk images</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QCheckBox" name="CH_Copy_Floppy">
       <property name="text">
        <string>Copy floppy images</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="CH_Copy_Hard_Drive">
       <property name="text">
        <string>Copy hard drive images</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="Dialog_Button_Box" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Dialog_Button_Box</class>
   <extends>QDialogButtonBox</extends>
   <header>Dialog_Button_Box.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>Edit_New_VM_Name</tabstop>
  <tabstop>CH_Copy_Disk_Images</tabstop>
  <tabstop>CH_Copy_Floppy</tabstop>
  <tabstop>CH_Copy_Hard_Drive</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>CH_Copy_Disk_Images</sender>
   <signal>toggled(bool)</signal>
   <receiver>CH_Copy_Floppy</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>82</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CH_Copy_Disk_Images</sender>
   <signal>toggled(bool)</signal>
   <receiver>CH_Copy_Hard_Drive</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Copy_VM_Window</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Copy_VM_Window</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
